test/upgrade/generate_cert.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+openssl req -x509 -newkey rsa:4096 -sha256 -days 3650 -nodes \
+  -keyout fleet.key -out fleet.crt -extensions san -config \
+  <(echo "[req]";
+    echo distinguished_name=req;
+    echo "[san]";
+    echo subjectAltName=DNS:fleet-a,DNS:fleet-b
+    ) \
+  -subj "/CN=fleet"
```

Comment:
Just curious: Any reason you had to generate a new pair instead of using the development ones? (`tools/osquery/fleet.crt` & `tools/osquery/fleet.key`)

Reply:
I asked the same question 😅 , got a great answer here https://github.com/fleetdm/fleet/pull/6596#discussion_r919400926